host/4/bullseye/amd64/java/java21/java21-build.Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM mcr.microsoft.com/dotnet/sdk:6.0 
```

Comment:
as bulleseye will be retiring soon, new images need to be on book work.

@CooperLink - is that possible?